api_app/db/repositories/airlock_requests.py

```diff
@@ -85,42 +85,43 @@ def validate_status_update(self, current_status: AirlockRequestStatus, new_statu
 
         return approved_condition and rejected_condition and blocked_condition and (approved_in_progress_condition or rejected_in_progress_condition or blocking_in_progress_condition or draft_condition or submit_condition or in_review_condition or cancel_condition or failed_condition)
 
-    def create_airlock_request_item(self, airlock_request_input: AirlockRequestInCreate, workspace_id: str) -> AirlockRequest:
+    def create_airlock_request_item(self, airlock_request_input: AirlockRequestInCreate, workspace_id: str, user) -> AirlockRequest:
         full_airlock_request_id = str(uuid.uuid4())
 
         resource_spec_parameters = {**self.get_airlock_request_spec_params()}
 
         airlock_request = AirlockRequest(
             id=full_airlock_request_id,
             workspaceId=workspace_id,
-            requestTitle=airlock_request_input.requestTitle,
+            title=airlock_request_input.title,
             businessJustification=airlock_request_input.businessJustification,
-            requestType=airlock_request_input.requestType,
-            creationTime=datetime.utcnow().timestamp(),
+            type=airlock_request_input.type,
+            createdBy=user,
```

Comment:
do we need to add `updatedBy` here, as we then copy that into the history?

Reply:
Again great spot you're absolutely right. Fixed!